app/service/ccd-service.ts

```diff
@@ -44,7 +44,8 @@ class CcdService {
       headers: {
         Authorization: headers.userToken,
         ServiceAuthorization: headers.serviceToken,
-        'content-type': 'application/json'
+        'content-type': 'application/json',
+        UserId: userId // Hack param to prove RIA-5761.
```

Comment:
@hmcts-version1-ignacio @addison-jephtah @rajeshthuraiyur  - I added this userid header param which is only required to get tests running.